Jenkinsfile

```diff
@@ -169,41 +192,38 @@ pipeline {
                             deleteDir()
                             unstash 'CIScripts'
                             powershell script: './CIScripts/Test.ps1'
+
+                            // TODO: you can do better
+                            stash name: 'testReport', includes: 'CIScripts/Test/testReport.xml'
```

Comment:
See my comment about testReport output path. Here we can see that there's really something wrong. Why would test report file be contained within CIScripts dir?